ui/tooltip.js

```diff
@@ -280,14 +280,28 @@ $.widget( "ui.tooltip", {
 		tooltipData = this._tooltip( target );
 		tooltip = tooltipData.tooltip;
 		this._addDescribedBy( target, tooltip.attr( "id" ) );
+		// Parse content into DOM remove attributes id and name
+		if ( typeof content === "string" || content.nodeType ) {
+			if ( typeof content === "string" ) {
+				tooltip_content_div = document.createElement("div");
+				tooltip_content_div.innerHTML = content;
+				dom_content = $(tooltip_content_div);
+			} else {
+				dom_content = $("<div>").html( content );
+			}
+			dom_content.find( "[id]" ).removeAttr( "id" );
+			dom_content.find( "[name]" ).removeAttr( "name" );
+			content = dom_content.html();
+		} else if ( content.jquery ) {
+			content.removeAttr( "id" ).find( "[id]" ).removeAttr( "id" );
+			content.removeAttr( "name" ).find( "[name]" ).removeAttr( "name" );
+		}
 		tooltip.find( ".ui-tooltip-content" ).html( content );
```

Comment:
Hey scott I am working on all the other fixes mentioned here an I read the jshint output but could you please further explain this comment. Based on my tests if you don't strip before putting the content into the tooltip then it will have already interfered with the other radio buttons on the page. what problems does the stripping cause with the content?